backend/src/v5/routes/users.js

```diff
@@ -272,6 +274,10 @@ const establishRoutes = () => {
 	 *                   type: string
 	 *                   description: The API key of the user
 	 *                   example: 23b61deadbba098fec517dc4fcc84d68
+	 *                 isSso:
```

Comment:
Either this response needs to return `sso: aad`, or we need the unlink endpoint to be in sso.js 

As it stands it's not future proof. If we start supporting google, the front end has no clue whether he needs to call aad/unlink, or google/unlink